bac_index.qmd

```diff
@@ -32,3 +32,32 @@ toc-depth: 4
     -   [KG3 -- Kaplan-Meier Graphs by Biomarker Subgroups](./graphs/kg03.qmd)
     -   [KG4 -- Kaplan-Meier Graphs by Treatment Arm and Biomarker Subgroups](./graphs/KG4/kg04.qmd)
     -   [KG5 -- Kaplan-Meier Graphs by Treatment Arm and Continuous Biomarker Cut](./graphs/KG5/kg05.qmd)
+-   [RFG -- Response Forest Graphs](./graphs/RFG1/rfg01.qmd)
```

Comment:
how about have sections in this index "Tables" and "Graphs" ? since we also have that split in the navigation pane

Reply:
Hi @danielinteractive: I did not move from the tables/graphs main folder all the templates that did not have A/B/etc variants, this is the case of dg04.qmd but also others, this aims at keep the structure and relative URLs simpler, if needed can be changed (requires revising all cross-links)
About the index, I actually see the same split like in the nav pane (maybe have a look to the rendered website), or do you mean anything else?
```
### Tables

-   [CMT -- Cox Proportional Hazards Model Tables](./tables/cmt01.qmd)
    -   [CMT1 -- Cox Proportional Hazards Model Tables with a Single Covariate](./tables/cmt01.qmd)
    -   [CMT2 -- Cox Proportional Hazards Model Tables with Multiple Covariates](./tables/CMT2/cmt02.qmd)
    -   [CMT3 -- Cox Proportional Hazards Model Tables with Separate Models for Each Covariate](./tables/cmt03.qmd)
-   [DT -- Demographics Tables](./tables/DT1/dt01.qmd)
    -   [DT1 -- Demographics Tables Comparing BEP vs. Overall Population](./tables/DT1/dt01.qmd)
    -   [DT2 -- Demographics Tables Comparing BEP vs. non-BEP Population](./tables/DT2/dt02.qmd)
-   [TET -- Time-to-Event Summary Tables](./tables/TET1/tet01.qmd)
    -   [TET1 -- Time-to-Event Summary Tables](./tables/TET1/tet01.qmd)

### Graphs

-   [AG -- Utilities for All Graphs](./graphs/ag01.qmd)```

Reply:
oh sorry, I overlooked the Tables and Graphs section titles... so all good!